src/scanner.h

```diff
@@ -90,7 +90,7 @@ class Scanner {
   bool InBlockContext() const { return m_flows.empty(); }
   std::size_t GetFlowLevel() const { return m_flows.size(); }
 
-  Token::TYPE GetStartTokenFor(IndentMarker::INDENT_TYPE type) const;
+  static Token::TYPE GetStartTokenFor(IndentMarker::INDENT_TYPE type);
```

Comment:
This method is only used in the .cpp file; can you just put it in an anonymous namespace there (rather than making it a static member function)?

Reply:
It uses INDENT_TYPE , which is declared private.